src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/IEPWarningService.kt

```diff
@@ -87,9 +85,6 @@ class IEPWarningService(
 
   private fun formatReasonAndComment(reason: AbsentReason, subReason: AbsentSubReason?, comment: String?): String {
     val caseNoteComment = subReason?.let { "${subReason.label}. $comment" } ?: comment
-    return when (reason) {
-      AbsentReason.RefusedIncentiveLevelWarning -> "Refused - Incentive Level warning - $caseNoteComment"
```

Comment:
This didn't really make much sense previously since the unacceptable absence also generated an incentive level warning but the code only put the comment on the refused.  I guess this was because there were two refused scenarios.  Anyway since the reason now knows how to format itself with a warning then better to just use that.